x-pack/platform/plugins/shared/task_manager/server/task_running/task_runner.ts

```diff
@@ -250,8 +251,7 @@ export class TaskManagerRunner implements TaskRunner {
    * @param id
    */
   public isSameTask(executionId: string) {
-    const executionIdParts = executionId.split('::');
```

Comment:
Do we have a constraint that a task id can't contain `::`, and I guess shouldn't start with a colon (not sure about that, maybe?).  I poked around a little and didn't see one, but feels like we should have one.

Reply:
Issue: https://github.com/elastic/kibana/issues/243230